examples/default-iam-user/README.md

```diff
@@ -1,16 +1,20 @@
 # Default Example
 
-This example creates an IAM user, group, and policy needed for Lacework integration/
+This example creates an OCI user, group, and policy needed for Lacework
+integration. The following example will use the OCI credentials in your
+~/.oci/config file. See the OCI Terraform Provider documentation [on
+authentication](https://docs.oracle.com/en-us/iaas/Content/API/SDKDocs/terraformproviderconfiguration.htm) for alternative approaches of providing
+credentials to the OCI provider.
 
 ```hcl
 module "lacework_iam_user" {
-  source = "../.."
+  source = "lacework/iam-user/oci"
```

Comment:
Not sure I see the diff between this example and the custom-aim-user example?

Reply:
Good point. custom-iam-user is supposed to be an example of using most/all of the variables. But it doesn't here. Let me update that.